pycmds/_record.py

```diff
@@ -779,17 +514,17 @@ def queue_control_update(self):
             self.set_freerun(True)
 
     def set_freerun(self, state):
-        for device in self.devices:
-            device.set_freerun(state)
+        for sensor in self.sensors:
+            sensor.set_freerun(state)
 
     def shutdown(self):
         self.set_freerun(False)
         self.wait_until_done()
-        for device in self.devices:
-            device.close()
+        for sensor in self.sensors:
+            sensor.close()
 
     def update_cols(self, aqn):
-        for cols_type in ["data", "shots"]:
+        for cols_type in ["data"]:
```

Comment:
I _think_ this loop can just be removed entirely (i.e. remove this line and dedent the contents)

Reply:
@untzag ping